### PR TITLE
CSIC-49 Provision and de-provision of treeq

### PR DESCRIPTION
**Pull request by gadekarnitesh**
Changes:

1. treeq treeq_path_prefix implementation

### CODE REVIEW

**Review by nikhilbarge — APPROVED** [review]
lg

File: deploy/examples/treeq/treeq-pvc.yaml [hunk 1]
```diff
@@ -1,14 +1,14 @@
 apiVersion: v1
 kind: PersistentVolumeClaim
 metadata:
-  name: ibox-tree-pvc-demo
+  name: ibox-tree-pvc-demo15
```
Comment (dhanashreebiradar) [inline deploy/examples/treeq/treeq-pvc.yaml]: remove 15

**Review by dhanashreebiradar — COMMENTED** [review]
Fix comment

**Review by dhanashreebiradar — APPROVED** [review]
lg

**Pull request merged**